wordpress/wordpress.go

```diff
@@ -260,3 +267,23 @@ loop:
 	}
 	return "", err
 }
+
+func removeInactives(elements models.WordPressPackages) models.WordPressPackages {
+	var inactives []int
+	for i, y := range elements {
+		if y.Status == "inactive" {
+			inactives = append(inactives, i)
+		}
+	}
+	for _, h := range inactives {
+		elements = removeAt(elements, h)
```

Comment:
This code looks like it does not work well. Slice index will change after `removeAt`.
Please add testcase to contain 2 or more `inactive` status packages.

Maybe following code works well.

```
var removed []models.WpPackage
for _, p := range pkgs {
  if p.Status == "inactive" {
    continue
  }
  removed = append(removed, p)
}
return removed
```

Reply:
fix it.